pages/api/create-event.js

```diff
@@ -34,7 +35,7 @@ async function storeFiles(files) {
 async function makeFileObjects(body) {
   const buffer = Buffer.from(JSON.stringify(body));
 
-  const imageDirectory = resolve(process.cwd(), "public/images/event.png");
+  const imageDirectory = resolve(process.cwd(), `public/images/${body.image}`);
```

Comment:
the subgraph currently looks for event.png ([reference](https://github.com/womenbuildweb3/web3RSVP-subgraph/blob/main/src/mapping.ts#L25)). it won't know what image file to look for since we're not storing that info  in the smart contract.